spec/classes/collectd_plugin_mcelog_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'spec_helper'
+
+describe 'collectd::plugin::mcelog', type: :class do
+  on_supported_os(baseline_os_hash).each do |os, facts|
+    context "on #{os} " do
+      let :facts do
+        facts
+      end
+
```

Comment:
can you please also ensure that this compiles properly?

Something like this should work:

```
it { is_expected.to compile.with_all_deps }
```

Reply:
Added